SimulationAgent.Test/DeviceProperties/DevicePropertiesActorTest.cs

```diff
@@ -72,22 +88,70 @@ public void Handle_Event_Should_Throw_When_Out_Of_Range()
                 () => this.target.HandleEvent(OUT_OF_RANGE_EVENT));
         }
 
+        [Fact, Trait(Constants.TYPE, Constants.UNIT_TEST)]
+        public void Should_Return_CountOfFailedTwinUpdates_When_TwinUpdateFails()
```

Comment:
Allow me to say that this naming convention is really weird. Typically one would choose between PascalCasing or snake_casing, not use both...

also why doesn't "CountOfFailedTwinUpdates" have the "_" ?

my suggestion would be to stick to PascalCasing, which is how we address readability everywhere

Reply:
That is because of me. I suggested we use Should_ExpectedBehavior_When_StateUnderTest naming scheme. I think it is easier to read and understand what exactly we are testing.